product_packaging_usability/tests/test_module.py

```diff
@@ -0,0 +1,16 @@
+from odoo.tests.common import TransactionCase
+
+
+class Test(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+    def test_action(self):
+        product = self.env.ref("product.product_product_6")
+        action = product.ui_goto_packaging_view()
```

Comment:
Great! Like it. We are so often too lazy to do kind of tests. :smiley:

Reply:
I'm lazy (permanently) and now I am ashamed to write this kind of test